README.md

```diff
@@ -29,7 +29,7 @@ cf restage $YOUR_APP_NAME
 
 #### Log Collection
 
-To start collecting logs from your application in CloudFoundry, the puppy-agent contained in the buildpack needs to be run and log collection enabled.
+To start collecting logs from your application in CloudFoundry, the puppy-agent contained in the buildpack needs to be run and log collection enabled. Note that DogStatsD is automatically enabled with the puppy-agent.
```

Comment:
puppy-agent isn't a user facing concept yet, no need to mention it here.   DogStatsD isn't enabled in the puppy-agent, but rather enabled in the Datadog for Application Monitoring for CloudFoundry build pack.